File: Beecrowd/1154 Idades/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        int x, cont = 0;
        double media = 0;

        do {
            x = sc.nextInt();

            if (x >= 0) {
                media += x;
                cont++;
            }

        } while (x >= 0);

        media /= cont;

        System.out.printf("%.2f%n", media);

        sc.close();
    }
}

File: Beecrowd/1154 Idades/main.c
#include <stdio.h>
 
int main() {
 
    int X, cont = 0;
	double media = 0;
	
	do{
	
	scanf("%d", &X);
	
	if(X >= 0){
		media += X;
		cont++;
	}	
	
	}while(X >= 0);
	
	media /= cont;
	
	printf("%.2lf\n", media);
 
    return 0;
}